onnxruntime/core/providers/webnn/builders/helper.h

```diff
@@ -188,6 +188,7 @@ static const InlinedHashMap<std::string, WebnnOpInfo> op_map = {
     {"GlobalLpPool", {"l2Pool2d", false}},
     {"Greater", {"greater", true}},
     {"GreaterOrEqual", {"greaterOrEqual", true}},
+    {"GRU", {"gru", false}},
```

Comment:
CPU backend now has supported `gru`, pls. set it to `true`.

Reply:
Done.